browser/mapping.go

```diff
@@ -468,11 +468,16 @@ func mapPage(vu moduleVU, p *common.Page) mapping {
 		"addStyleTag":   p.AddStyleTag,
 		"bringToFront":  p.BringToFront,
 		"check":         p.Check,
-		"click": func(selector string, opts goja.Value) *goja.Promise {
+		"click": func(selector string, opts goja.Value) (*goja.Promise, error) {
+			popts, err := parseFrameClickOptions(vu.Context(), opts, p.Timeout())
```

Comment:
nit: s/popts/clickOpts

Reply:
I'll do a mass renaming later if we need. Sticking with popts for now.